allocation.md

```diff
@@ -6,379 +6,107 @@ Throughout our API documentation, we use `localhost:9090` as the default Kubecos
-
-`namespace`
-
-, 
-
-`controller`
-
-, and 
-
-`label`
-
-. Data is served from one of 
-
-[Kubecost's ETL pipelines](cost-model-deprecated.md#caching-overview)
-
-.
+The Allocation API is the preferred way to query for costs and resources allocated to Kubernetes workloads and optionally aggregated by Kubernetes concepts like `namespace`, `controller`, and `label`. Data is served from one of [Kubecost's ETL pipelines](cost-model-deprecated.md#caching-overview).
 {% endswagger-description %}
 
 {% swagger-parameter in="path" name="window" type="string" required="true" %}
-Duration of time over which to query. Accepts words like 
-
-`today`
-
-, 
-
-`week`
-
-, 
-
-`month`
-
-, 
-
-`yesterday`
-
-, 
-
-`lastweek`
-
-, 
-
-`lastmonth`
-
-; durations like 
-
-`30m`
-
-, 
-
-`12h`
-
-, 
-
-`7d`
-
-; comma-separated RFC3339 date pairs like
-
-`2021-01-02T15:04:05Z,2021-02-02T15:04:05Z`
-
-; comma-separated Unix timestamp (seconds) pairs like 
-
-`1578002645,1580681045`
-
-.
+Duration of time over which to query. Accepts words like `today`, `week`, `month`, `yesterday`, `lastweek`, `lastmonth`; durations like `30m`, `12h`, `7d`; comma-separated RFC3339 date pairs like `2021-01-02T15:04:05Z,2021-02-02T15:04:05Z`; comma-separated Unix timestamp (seconds) pairs like `1578002645,1580681045`.
 {% endswagger-parameter %}
 
 {% swagger-parameter in="path" name="aggregate" type="string" required="false" %}
-Field by which to aggregate the results. Accepts: 
-
-`cluster`
-
-, 
-
-`namespace`
-
-, 
-
-`controllerKind`
-
-, 
-
-`controller`
-
-, 
-
-`service`
-
-, 
-
-`node`
-
-, 
-
-`pod`
-
-, 
-
-`label:<name>`
-
-, and 
-
-`annotation:<name>`
-
-. Also accepts comma-separated lists for multi-aggregation, like 
-
-`namespace,label:app`
-
-.
+Field by which to aggregate the results. Accepts: `cluster`, `namespace`, `controllerKind`, `controller`, `service`, `node`, `pod`, `label:<name>`, and `annotation:<name>`. Also accepts comma-separated lists for multi-aggregation, like `namespace,label:app`.
 {% endswagger-parameter %}
 
 {% swagger-parameter in="path" name="accumulate" type="boolean" required="false" %}
-If 
-
-`true`
-
-, sum the entire range of sets into a single set. Default value is 
-
-`false`
-
-.
+If `true`, sum the entire range of sets into a single set. Default value is `false`.
 {% endswagger-parameter %}
 
 {% swagger-parameter in="path" name="idle" type="boolean" required="false" %}
-If 
-
-`true`
-
-, include idle cost (i.e. the cost of the un-allocated assets) as its own allocation. (See 
-
-[special types of allocation](https://docs.kubecost.com/apis/apis-overview/allocation#special-types-of-allocation)
-
-.) Default is 
-
-`true.`
+If `true`, include idle cost (i.e. the cost of the un-allocated assets) as its own allocation. (See [special types of allocation](https://docs.kubecost.com/apis/apis-overview/allocation#special-types-of-allocation).) Default is `true.`
```

Comment:
You may want to take opportunities like this one to change these to relative links.

Reply:
Will do this in a separate PR